### PR TITLE
DATAUP-494: table layout changes

### PR DESCRIPTION
**Pull request by ialarmedalien**
Screenshot 2021-09-23 at 10 26 32

# Description of PR purpose/changes

Job Status Table layout changes:
- removed status details column
- added import type and output columns

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-494
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative, creating and running a bulk import cell, and then checking out the job status table

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 1]
```diff
@@ -42,27 +44,31 @@ define([
                             [
                                 th(
                                     {
-                                        class: `${cssBaseClass}__table_head_cell--object`,
+                                        class: `${cssBaseClass}__table_head_cell--import-type`,
+                                        style: 'width: 25%',
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:48]: I hate having to do this, but it seems the only way* to stop DataTables assigning an absolute width to the table.

* I haven't yet investigated if there is an official way to do it. Going to do that now.

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 2]
```diff
@@ -152,17 +212,35 @@ define([
...
+                                        : row.job_id;
 
-                                if (row.retry_parent && jobInfo[row.retry_parent]) {
-                                    return renderParams(jobInfo[row.retry_parent]);
+                                if (jobParams[relevantJobId]) {
+                                    return displayAppType(jobParams[relevantJobId].analysisType);
                                 }
-                                if (jobInfo[row.job_id]) {
-                                    return renderParams(jobInfo[row.job_id]);
+                                return `Job ID: ${row.job_id}`;
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:227]: show the job ID in the leftmost row if the import type has not yet been established

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 3]
```diff
@@ -416,10 +480,19 @@ define([
          * @param {object} message
          */
         function handleJobInfo(_, message) {
-            const { jobId } = message;
+            const { jobId, jobInfo } = message;
             const jobState = jobManager.model.getItem(`exec.jobs.byId.${jobId}`);
+            const appData = jobManager.model.getItem('app');
             const rowIx = jobState && jobState.retry_parent ? jobState.retry_parent : jobId;
 
+            const jobDisplayData = generateJobDisplayData({
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:488]: when the job info comes in, create the job display data

File: test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js [hunk 4]
```diff
@@ -165,17 +271,21 @@ define([
         // row ID: may be specified in job.meta.row_id, e.g. if the job is a retry
         expect(row.id).toEqual(`job_${job.meta && job.meta.row_id ? job.meta.row_id : job.job_id}`);
 
-        // status details cell
-        const logViewEl = row.querySelector(`.${cssBaseClass}__cell--log-view`);
-        expect(logViewEl.textContent).toContain('Status details');
+        // type
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js:274]: no more status details cell; now have the import type and the output object name

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 5]
```diff
@@ -28,6 +29,7 @@ define([
             {
                 class: `${cssBaseClass}__table`,
                 caption: 'Job Status',
+                style: 'width: 100%',
```
Comment (briehl) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:32]: Should this just go in the scss file under `kb-job-status__table` ?

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js (hunk 1 above)
Comment (briehl) [reply kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:48]: Ok. That's fine. Datatables has some fun idiosyncrasies around its styling.

Comment (briehl) [reply kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:48]: You might be able to get around it if you declare all the column headers in the DataTables call.

**Review by briehl — APPROVED** [review]

**Pull request merged**